@kiva/kv-components/package.json

```diff
@@ -46,6 +46,8 @@
     "@kiva/kv-tokens": "^0.7.0",
     "@mdi/js": "^5.9.55",
     "aria-hidden": "^1.1.3",
+    "embla-carousel": "^4.5.3",
+    "lodash": "^4.17.21",
```

Comment:
anyway we can skip this dependency? Seems like we could come up with a lighter solution for this one use. _If not let's please make sure the tree shaking is happening._

Reply:
Thank you!